docs/arlas-api-exploration.md

```diff
@@ -163,7 +163,7 @@ Of course, you can combine both way to handle complex multiple filters for each
 | **`:gt:`**    | `{fieldName}` is greater than `{value}`                                                                             | numeric            |
 | **`:lte:`**   | `{fieldName}` is less than or equal to `{value}`                                                                    | numeric            |
 | **`:lt:`**    | `{fieldName}` is less than `{value}`                                                                                | numeric            |
-| **`:range:`** | `{fieldName}` is between `{comma separated [min<max] values}`. **OR** operation is applied for the specified ranges | numeric or strings. If the field's type is date, min & max should be timestamps in millisecond |
+| **`:range:`** | `{fieldName}` is between `{comma separated [min<max] values}`. **OR** operation is applied for the specified ranges | numeric or strings. If the field's type is date, min & max should be timestamps in millisecond or [Date Math expression](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math) |
```

Comment:
I would rather use our own terminology, specify it's actually the same as `Date Math Expression` and keep a link to elasticsearch documentation.